jbake/content/cas.adoc

```diff
@@ -0,0 +1,179 @@
+[#CAS-IntegratingApacheShirowithCASSSOserver]
+= Integrating Apache Shiro with CAS SSO server
+:jbake-type: page
+:jbake-status: published
+:jbake-tags: documentation, cas
```

Comment:
```suggestion
:jbake-tags: documentation, cas
:jbake-related: { "links": ["web-features.html"] }
```

Not strictly required, and I'm still not sure this is the _best_ way to handle these "related" sidebar section links.
(feel free to ignore 😄)